src/compiler/Compiler.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ *
+ * @flow
+ */
+
+import type { Platform, Logger } from '../types';
+
+const EventEmitter = require('events');
```

Comment:
We're using `import` in various other places. Can we change them to `require` or change these to `import` for consistency?

Reply:
Does importing a type with `require` works for u? And Haul is using `requires` everywhere.

Reply:
@satya164 let's do this in one PR with a codemod, no need to mess this now

Reply:
@zamotany no, will need import 🤔
@thymikee okay